c_src/device/drm.c

```diff
@@ -740,54 +764,42 @@ void device_end_render() {
   g_egl_data.frame_idx = next_idx;
 }
 
-
-
-
-void device_poll() {}
+void device_poll()
+{
+}
 
 // these case are factored out mostly so that they can be driven by different
 // GL includes as appropriate
-void device_clear() {
-  glClear( GL_COLOR_BUFFER_BIT );
+void device_clear()
+{
+  glClear(GL_COLOR_BUFFER_BIT);
 }
 
-void device_clear_color( float red, float green, float blue, float alpha ) {
-  glClearColor( red, green, blue, alpha );
+void device_clear_color(float red,
+                        float green,
+                        float blue,
+                        float alpha)
+{
+  glClearColor(red, green, blue, alpha);
 }
 
-char* device_gl_error() {
+char* device_gl_error()
+{
   GLenum err  = glGetError();
   switch (err)
   {
-    case GL_NO_ERROR:
-      return NULL;
-
-    case GL_INVALID_ENUM:
-      return "GL_INVALID_ENUM";
-
-    case GL_INVALID_VALUE:
-      return "GL_INVALID_VALUE";
-
-    case GL_INVALID_OPERATION:
-      return "GL_INVALID_OPERATION";
-
-    case GL_OUT_OF_MEMORY:
-      return "GL_OUT_OF_MEMORY";
-
+  case GL_NO_ERROR: return NULL;
+  case GL_INVALID_ENUM: return "GL_INVALID_ENUM";
+  case GL_INVALID_VALUE: return "GL_INVALID_VALUE";
+  case GL_INVALID_OPERATION: return "GL_INVALID_OPERATION";
+  case GL_OUT_OF_MEMORY: return "GL_OUT_OF_MEMORY";
 #ifdef GL_STACK_UNDERFLOW
-    case GL_STACK_UNDERFLOW:
-      return "GL_STACK_UNDERFLOW";
+  case GL_STACK_UNDERFLOW: return "GL_STACK_UNDERFLOW";
 #endif
-
 #ifdef GL_STACK_OVERFLOW
-    case GL_STACK_OVERFLOW:
-      return "GL_STACK_OVERFLOW";
+  case GL_STACK_OVERFLOW: return "GL_STACK_OVERFLOW";
 #endif
-      
-    case GL_INVALID_FRAMEBUFFER_OPERATION:
-      return "GL_INVALID_FRAMEBUFFER_OPERATION";
-
-    default:
-      return "GL_OTHER";
+  case GL_INVALID_FRAMEBUFFER_OPERATION: return "GL_INVALID_FRAMEBUFFER_OPERATION";
+  default: return "GL_OTHER";
   }
 }
```

Comment:
This routine really wants to live in like a `gl_helpers.c` file.

Reply:
Moved to `c_src/device/gl_helpers.c`